client/my-sites/themes/theme-showcase.jsx

```diff
@@ -15,11 +15,10 @@ import QueryProductsList from 'calypso/components/data/query-products-list';
 import QuerySitePlans from 'calypso/components/data/query-site-plans';
 import QuerySitePurchases from 'calypso/components/data/query-site-purchases';
 import QueryThemeFilters from 'calypso/components/data/query-theme-filters';
-import { SearchThemes, SearchThemesV2 } from 'calypso/components/search-themes';
```

Comment:
It's safe to delete this `SearchThemesV2` component.

Reply:
Within this change, we use the same Search control for logged-in and logged-out users, so we don't need two versions.

Buuut, when I think more, maybe that's why the SSR test doesn't work. Maybe the V2 should simply render the component without extra logic. I'll try that scenario.

Reply:
Sorry, it isn't clear whether we can remove the unnecessary file anymore. Can we?

Reply:
You are right. I forgot to remove the component. It's removed now.
Thanks for the feedback.